Views/Cart/Confirmation.cshtml

```diff
@@ -0,0 +1,32 @@
+@model BangazonWeb.ViewModels.CartView
+@using Bangazon.Models;
+
+@{
+    ViewData["Title"] = "Shopping Cart";
+}
+
+<h1>Thank you for your Order! (@Model.OrderId)</h1>
+
```

Comment:
Maybe be a little more clear about the order number, like
`Thank you for your Order #12345!`
and then show them the payment method they used as well